src/app/modules/rup/components/elementos/adjuntarDocumento.component.ts

```diff
@@ -65,6 +66,14 @@ export class AdjuntarDocumentoComponent extends RUPComponent implements OnInit {
         });
     }
 
+    onValidate() {
+        if (!this.params) {
```

Comment:
Acá falta el nombre del parámetro, a futuro se puede agregar otros parámetros a este átomo. 
```suggestion
        if (!this.params.required) {
```